pkg/api/latest/latest.go

```diff
@@ -28,17 +28,25 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 )
 
-// Version is the string that represents the current external default version.
+// GroupVersion is the string that represents the current external default groupVersion.
+var GroupVersion string
+
+//
+var Group string
+
+//Version is the string that represents the current external default Version.
 var Version string
```

Comment:
We're going to need an interface/type for the set of information associated with a group and/or groupVersion: default version, oldest version, storage version, etc.